python/lsst/meas/base/base.py

```diff
@@ -39,6 +39,26 @@
 # Exceptions that the measurement tasks should always propagate up to their callers
 FATAL_EXCEPTIONS = (MemoryError, FatalAlgorithmError)
 
+# Set of names of algorithms that measure fluxes that can be aperture corrected
+_ApCorrNameSet = set()
+
+def addApCorrName(name):
+    """!Add to the set of names of algorithms that measure fluxes that can be aperture corrected
+
+    @param[in] name  name of algorithm that measures a flux that can be aperture corrected,
+        such as "slot_PsfFlux"
+    """
+    global _ApCorrNameSet
+    _ApCorrNameSet.add(str(name))
+
+def getApCorrNameSet():
+    """!Return a copy of the set of names of algorithms that measure fluxes that can be aperture corrected
+
+    For example the returned set will likely include "slot_PsfFlux" and "slot_GaussianFlux"
+    """
+    global _ApCorrNameSet
+    return _ApCorrNameSet.copy()
+
```

Comment:
The names you're using as examples here (`slot_PsfFlux`, `slot_GaussianFlux`) aren't actually algorithm names - `slot_PsfFlux` is the aliase created to identify which algorithm is in the PsfFlux slot, and `slot_GaussianFlux` doesn't actually exist.  The names of the algorithms you're thinking of are `base_PsfFlux` and `base_GaussianFlux` (the first component of the name indicates the package the algorithm lives in; "base" is for "meas_base").